relay-event-normalization/src/stacktrace.rs

```diff
@@ -41,6 +42,33 @@ pub fn process_non_raw_frame(frame: &mut Annotated<Frame>) {
                 }
             }
         }
+
+        if frame.function.as_str() == Some("?") {
```

Comment:
The order is the same as in the processor: process the stack trace first (code above this +diff), then process frames (this +diff).

Reply:
The call to `process_child_values` is gone here though. Are we sure that was a noop?

Reply:
There are no `process_*` calls for a `Frame`'s properties, so these aren't further normalized.